### DISCUSSION TITLE
График, установка мин, макс, шаг для осей

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#363] [Как это сделать?] [answered]
Почему не работает фиксированная установка мин, макс шаг для осей?

![image](https://user-images.githubusercontent.com/114401808/235835550-edc27f4a-21c3-4ada-9989-a5894237bc87.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Если чесн - фиг его знает. Может мы обработку параметров осей не подключили или эта настройка отн к другой версии chartjs

**Reply by vldekov** [reply]
Какая версия chartjs стоит в тотум? Возможно её обновить до текущей версии?

**Reply by ttm-alexeypolunin** [reply]
Вторая, нет
